common/src/main/resources/assets/meadow/lang/en_us.json

```diff
@@ -330,4 +330,4 @@
   "tooltip.meadow.thankyou_3": "Thank you for playing Meadow!",
   "tooltip.meadow.thankyou_4": "Grants Dig Speed II in a radius of 8 blocks",
   "item.meadow.meadow_standard": "Completionist Banner: §2Meadow"
-}
\ No newline at end of file
+}
```

Comment:
I have no idea what happened on line 333. I hardly understand how a pull request works. I was only trying to fix line 157.